vhost-device-vsock/src/main.rs

```diff
@@ -208,7 +209,7 @@ fn parse_vm_params(s: &str) -> Result<VsockConfig, VmArgsParseError> {
                 guest_cid = Some(val.parse().map_err(VmArgsParseError::ParseInteger)?)
             }
             "socket" => socket = Some(val.to_string()),
```

Comment:
```suggestion
            "socket" => socket = Some(PathBuf::from(val.to_string())),
```

So we can remove the change on line 267.